ignition/modules/amm.ts

```diff
@@ -1,7 +1,7 @@
 import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
```

Comment:
Do we need this now if we have a task for it?

Reply:
removed